File: src/main/resources/output/IFC4X3_csharp/Entities/IfcStructuralLoad/IfcStructuralLoadOrResult/IfcStructuralLoadStatic/IfcStructuralLoadSingleForce/IfcStructuralLoadSingleForce.cs
public class IfcStructuralLoadSingleForce : IfcStructuralLoadStatic
{
    public IfcForceMeasure ForceX { get; set; }
    public IfcForceMeasure ForceY { get; set; }
    public IfcForceMeasure ForceZ { get; set; }
    public IfcTorqueMeasure MomentX { get; set; }
    public IfcTorqueMeasure MomentY { get; set; }
    public IfcTorqueMeasure MomentZ { get; set; }

    // === EXTENDED BY ===
    // IfcStructuralLoadSingleForceWarping
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcStructuralLoad/IfcStructuralLoadOrResult/IfcStructuralLoadStatic/IfcStructuralLoadSingleForce/IfcStructuralLoadSingleForce.py
class IfcStructuralLoadSingleForce(IfcStructuralLoadStatic):
    def __init__(self):
        self.ForceX: IfcForceMeasure = None
        self.ForceY: IfcForceMeasure = None
        self.ForceZ: IfcForceMeasure = None
        self.MomentX: IfcTorqueMeasure = None
        self.MomentY: IfcTorqueMeasure = None
        self.MomentZ: IfcTorqueMeasure = None

    # === EXTENDED BY ===
    # IfcStructuralLoadSingleForceWarping
